### ISSUE TITLE
qbittorrent cannot connect

### ISSUE THREAD

**Issue by beyondmeat** [jesec/flood#388]
Type: Bug Report

- [ ] Try to follow the update procedure described in the README and try again before opening this issue.

## Your Environment

- Version used:   jesec/flood:master
  - Version: 4.6.1
  - Commit ID: Flood 0.0.0-master.b67f825
- Environment name and version:
  - Node.js version `node --version`
  - npm version `npm --version`
  - Web browser `name and version`
- Operating system and version: Docker on Ubuntu
- Torrent client and version: qbittorrent 4.3.7

## Summary

```GET /api/client/settings 500 11.822 ms - 102
Error
    at clientRequestManager_ClientRequestManager.updateAuthCookie (/usr/local/lib/node_modules/@jesec/flood/dist/index.js:438:143087)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
...
Error
    at clientRequestManager_ClientRequestManager.updateAuthCookie (/usr/local/lib/node_modules/@jesec/flood/dist/index.js:438:143087)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
GET /api/activity-stream?historySnapshot=FIVE_MINUTE 200 2.924 ms - -
```

## Expected Behavior

works

## Current Behavior

cannot connect to client

## Steps to Reproduce

1. setup flood with qbittorrent
2. try to connect

## Context

Configured it right, was working, but stopped working sometime in June.

```
    command:
      - --rundir=/config
      - --auth=none
      - --qburl=http://qbittorrent:8080
      - --qbuser=admin
      - --qbpass=adminadmin
 ```

**Comment by beyondmeat** [comment]
I changed the webui to VueTorrent and now flood works. So it seems like the default webui may have changed something that broke this.

**Comment by jesec** [comment]
You probably used a wrong password initially. qBittorrent has an extremely conservative anti-brute-force threshold (something like 3 failures and you are locked out for a day). Flood's auto reconnect mechanisms, unfortunately, will trigger the rate limit very quick.

A reboot of qBittorrent is needed to reset the limit, and I suspect that is what truly fixed your problem. Note that web UIs never change the behavior of API endpoints of qBittorrent.

**Comment by beyondmeat** [comment]
No that's not what it was. I use the default password since it's not exposed to the outside world and never changed the default. I update my docker containers nightly and reboot a lot. I restarted both containers a few times as well. Definitely not that at all. I have the click jacking and other stuff already disabled.

I wiped out the flood config folder and removed the commands and reran the container. Inputted the connection settings manually, still couldn't connect. Restored the commands.

Only worked when I switched the web UI to VueTorrent in settings and that was the only setting changed. it was an unexpected surprise when flood was connected again, as I didn't do that to fix flood.

I suspect doing that though did something to another webui setting perhaps because I turned off vuetorrent, wiped /config for flood, restarted both containers and it connected fine. So I suspect it was a weird config state that was causing it. Since I can't repro anymore I'll close.
